reference_data/centos7.yml

```diff
@@ -0,0 +1,116 @@
+    packages_git: 'https://git.centos.org/projects/rpms/%2A'
+    #? what tag to use here ?#
+    git_tag_prefix:
+      modified: a8
+      non_modified: c8
+    versions:
+      - name: '7.0'
+        version_prefix: '70000'
+        dist_prefix: 'el7.0.0'
+      - name: '7.1'
+        version_prefix: '70100'
+        dist_prefix: 'el7.1.0'
+      - name: '7.2'
+        version_prefix: '70200'
+        dist_prefix: 'el7.2.0'
+      - name: '7.3'
+        version_prefix: '70300'
+        dist_prefix: 'el7.3.0'
+      - name: '7.4'
+        version_prefix: '70400'
+        dist_prefix: 'el7.4.0'
+      - name: '7.5'
+        version_prefix: '70500'
+        dist_prefix: 'el7.5.0'
+      - name: '7.6'
+        version_prefix: '70600'
+        dist_prefix: 'el7.6.0'
+      - name: '7.7'
+        version_prefix: '70700'
+        dist_prefix: 'el7.7.0'
+      - name: '7.8'
+        version_prefix: '70800'
+        dist_prefix: 'el7.8.0'
+       - name: '7.9'
+        version_prefix: '70900'
+        dist_prefix: 'el7.9.0'
+
+  type: rpm
+  
+  arch_list:
+    - x86_64
+    - aarch64
+  copy_priority_arches:
+    - x86_64
+    - aarch64
+  #? I ve removed weak_arch_list since we dont work with those architectures ?# 
+  data:
+    definitions:
+      distribution: CentOS7
+      #? is this teamname/email correct ?#
+      packager: CentOS Packaging Team <packager@centos.org>
+      vendor: CentOS
+    mock:
+      chroot_setup_cmd: 'install bash bzip2 coreutils cloudlinux-release '
+                         'findutils gawk grep gzip info redhat-rpm-config '
+                            'sed shadow-utils tar unzip util-linux-ng which '
+                            'xz yum git'
+      #? I now sure how it will work since CentOS7 uses yum
+      dnf_common_opts:
+        - --setopt=deltarpm=False
+        - --allowerasing
+        - --nobest
+        - --setopt=install_weak_deps=False
+      dnf_install_command: install dnf dnf-plugins-core shadow-utils
+      package_manager: yum
+      releasever: '7'
+      environment:
+        DFLTCC: '0'
+      use_bootstrap_container: false
+      macros:
+        "%__pesign_cert": "%pe_signing_cert"
+        "%__pesign_client_cert": "%pe_signing_cert"
+        "%__pesign_client_token": "%pe_signing_token"
+        "%__pesign_token": "-t %pe_signing_token"
+        "%pe_signing_cert": "'Cloud Linux Software, Inc'"
+        "%pe_signing_token": "'profil standardowy'"
+        "%_pesign": "/usr/local/bin/pesign"
+        "%_pesign_client": "/usr/local/bin/pesign-client"
+        "%with_modsign": "1"
+        "%modsign_os": "centos7"
+    mock_dist: el7
+    timeout: 43200
+    yum:
+      best: false
+      module_platform_id: platform:el7
+  #? There are no repos for aarch64 ?#
+  repositories:
+    - arch: x86_64
+      name: centos-7-baseos
+      type: rpm
+      remote_url: http://mirror.centos.org/centos/7/os/x86_64/
+      export_path: centos/7/os/x86_64/
+      production: true
```

Comment:
These are not "production" repos by our standards, since we do not export data into them, so `production` should be `False` and `export_path` is not needed